src/emc/motion/mot_priv.h

```diff
@@ -160,6 +160,11 @@ typedef struct {
     hal_bit_t   *spindle_is_oriented;	/* in: orientation completed */
     hal_s32_t   *spindle_orient_fault;	/* in: error code of failed operation */
 
+    // spindle increase / decrease parameters
+    hal_u32_t spindle_speed_change_step_abs;	/* in: absolute step to increase or decrease spindle speed */
```

Comment:
Params have actually been deprecated in all instantiable components, albeit are supported for legacy code.
We are slowly phasing them out elsewhere, so would not want any new ones being created.

They came about because originally pins could not be set to default values, but now they can, a pin would serve as well and could be set internally or externally.